packages/itwinui-css/src/text/mixins.scss

```diff
@@ -31,4 +31,5 @@
   $type: if(map.has-key($sizes, $type), $type, body);
   font-size: map.get($sizes, $type);
   font-weight: map.get($weights, $type);
+  overflow-wrap: anywhere;
```

Comment:
Does `word-break: break-word` + `hyphens: auto` work instead of `overflow-wrap: anywhere`? I think that would be safer / have fewer undesirable results.